docs/pages/file-format.md

```diff
@@ -22,6 +22,24 @@ Archive the contents of a document folder with `zip`.
 zip -r -X document.sketch *
 ```
 
+## JSON Schema
+
+The [JSON Schema](https://github.com/sketch-hq/sketch-file-format) based specification describes how Sketch documents are stored on disk, and how this changes from version to version.
+
+We hope this helps everyone building low level integrations of Sketch, such as server and command line applications that create and maniupulate Sketch files outside the Mac app.
+
+## Typescript types
+
+The TypeScript [types](https://github.com/sketch-hq/sketch-file-format-ts) model Sketch file JSON, and are automatically generated from the file format [json schema](/file-format/reference/json-schema) specification.
+
+The types will allow you to strongly type representations and fragments of Sketch documents in TypeScript projects.
```

Comment:
Similar, more direct wording making it clear that it has types for all fragments such as artboards, layers, etc.